docs/docs-content/vertex/system-management/add-registry.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: "Add a Registry"
+title: "Add a Registry"
+description: "Learn how to add a registry in Palette VerteX."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+tags: ["vertex", "management", "registry"]
+---
+
+You can add a registry at the system level or tenant level. Registries added at the system level are available to all the tenants. Registries added at the tenant level are available only to that tenant. This section describes how to add a system-level registry. For guidance, check out [Add a Tenant-Level Registry](../../tenant-settings/add-registry.md).
+
+## Prerequisites
+
+- You need tenant admin privileges. 
+
+## Add an OCI Registry
+
+Use the following steps to add a system-level OCI registry.
+
+1. Log in to the Palette VerteX system console. Refer to [Access the System Console](system-management.md#access-the-system-console) guide.
+
+2. From the left **Main Menu** select **Administration**. 
+
+3. Select the **Pack Registries** tab and click on the **Add New Pack Registry** button.
+
+4. Fill out the following input values to add an OCI registry.
+
+  | **Field**            | **Description**                   |
+  |----------------------|-----------------------------------|
+  | **Name**                 |  A custom name for the registry.  |
+  | **Registry Type**        |  Select **OCI**.                  |
+  | **OCI Authentication Type**  |  Select **ECR**. Our OCI packs are hosted in the Amazon Elastic Container Registry (ECR). |
+  | **Endpoint**             |  Provide the endpoint `415789037893.dkr.ecr.us-east-1.amazonaws.com` and prefix it with `https://`. |
```

Comment:
```suggestion
  | **Endpoint**             |  Provide the URL to the registry endpoint.  |
```
We can't assume the user is using our registry.

Reply:
Added.